woocommerce/google-listings-and-ads
# Streamline onboarding: Move FAQs to the bottom of pages

### Changes proposed in this Pull Request:



Closes #2488

- Add new `StepContentActions` component which replaces `StepContentFooter`.
- Add a new `StepContentFooter` component which uses the existing `Section` and `VerticalGapLayout` components.
- Added the new `xlarge` size to `VerticalGapLayout` that uses the `--large-gap` CSS property for spacing.

### Screenshots:


**Onboarding flow:**
- ![image](https://github.com/user-attachments/assets/fc45c26a-f8a1-426e-b10f-84b9a54a4803)
- ![image](https://github.com/user-attachments/assets/62db10a3-443d-4639-8a50-b46fe16c52c8)

**Editing a paid campaign**
![image](https://github.com/user-attachments/assets/79556303-f9c7-4c25-8c7b-ee4b88b03080)

**Optimizing your campaign**
![image](https://github.com/user-attachments/assets/555b459b-bef8-4d87-ab22-cac560d99f6a)


### Detailed test instructions:

* Go through the onboarding flow.
* The FAQ section should be after the CTA for the following pages:
	* During the onboarding process, the first step ("Set up your accounts") and fourth step ("Create a campaign")
	* When editing a paid campaign (Go to the dashboard and "Edit" any paid campaign which has been previously created)
	* When optimizing a paid campaign.
		* Click on the "Add paid campaign" from the dashboard.
		* Fill in the form in the first step and go to the next one (Optimize your campaign) to see the FAQ.
* The spacing between the CTA and the FAQ panel should be 40px which is the value of the `--large-gap` CSS property as per the AC.

### Additional details:
On the first step of the onboarding process, there is extra space between the CTA and the section above because of the long text as per the screenshot:
![image](https://github.com/user-attachments/assets/d6fa0a5c-ec0f-4b0f-b9f2-53c8c4d31d57)


### Changelog entry

> Tweak - Move FAQs to the bottom of pages.
